### ISSUE TITLE
Image problems

### ISSUE THREAD

**Issue by DoctorDiLauro** [DoctorDiLauro/grid-layout#1]
Seems that the images provided are a little too big in comparison to the ones used for the screenshots, so it's setting Markbot off saying it looks too different.

@melissaroy

**Comment by melissaroy** [comment]
Yes it was a small mistake on our side, crop the image or you can download the ZIP of the repo and grab the updated images.
